wait-for-deployment

```diff
@@ -45,20 +45,20 @@ get_updated_replicas() {
   get_deployment_jsonpath '{.status.updatedReplicas}'
 }
 
-get_available_replicas() {
-  get_deployment_jsonpath '{.status.availableReplicas}'
+get_ready_replicas() {
+  get_deployment_jsonpath '{.status.readyReplicas}'
```

Comment:
It's because readyReplicas is available for both deployment and stateful sets and availableReplicas only for deployments. So using readyReplicas allows us checking deployments and stateful sets the same way.